config/jobs/kubernetes/sig-testing/bazel-build-test.yaml

```diff
@@ -144,7 +145,38 @@ postsubmits:
         resources:
           requests:
             memory: "6Gi"
+  - name: ci-kubernetes-bazel-build-1-12
+    branches:
+    - release-1.12
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+    spec:
+      containers:
+        # TODO(cjwagner): Change to 1.12 image tag when available.
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20180730-8b7ab3104-master
```

Comment:
you should be able to use the 1.12 images pushed by @amwat now

Reply:
Done.